CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 # UNRELEASED
 
+* TeamsComplianceREcordingPolicy
```

Comment:
Could you update your branch with Dev and then place this item alphabetically on the right place? Just the MISC and DEPENDENCIES ones should be at the bottom, the rest alphabetically.